src/gpio.rs

```diff
@@ -729,208 +669,208 @@ gpio!([
+                AF9: (into_af9, af9, ["stm32f373", "stm32f378",],),
+                AF11: (into_af11, af11, ["stm32f303xb", "stm32f303xc", "stm32f303xd", "stm32f303xe", "stm32f358", "stm32f398",],),
+                AF13: (into_af13, af13, ["stm32f334",],),
+            ]),
+            PA11: (pa11, 11, Input<Floating>, moder11, AFRH, afrh11, bs11, br11, odr11, idr11, pupdr11, ot11, [
+                AF6: (into_af6, af6,),
+                AF7: (into_af7, af7,),
+                AF15: (into_af15, af15,),
+            ], [
+                AF2: (into_af2, af2, ["stm32f373", "stm32f378",],),
+                AF5: (into_af5, af5, ["stm32f301", "stm32f303xd", "stm32f303xe", "stm32f318", "stm32f373", "stm32f378", "stm32f398",],),
+                AF8: (into_af8, af8, ["stm32f302", "stm32f303xb", "stm32f303xc", "stm32f303xd", "stm32f303xe", "stm32f373", "stm32f378", "stm32f358", "stm32f398",],),
+                AF9: (into_af9, af9, ["stm32f302", "stm32f303", "stm32f373", "stm32f378", "stm32f334", "stm32f328", "stm32f358", "stm32f398",],),
+                AF10: (into_af10, af10, ["stm32f302", "stm32f303xb", "stm32f303xc", "stm32f303xd", "stm32f303xe", "stm32f373", "stm32f378", "stm32f358", "stm32f398",],),
+                AF11: (into_af11, af11, ["stm32f301", "stm32f318", "stm32f302", "stm32f303", "stm32f334", "stm32f328", "stm32f358", "stm32f398",],),
+                AF12: (into_af12, af12, ["stm32f301", "stm32f318", "stm32f302", "stm32f303", "stm32f334", "stm32f328", "stm32f358", "stm32f398",],),
+                AF13: (into_af13, af13, ["stm32f334",],),
+                AF14: (into_af14, af14, ["stm32f302", "stm32f303xb", "stm32f303xc",],),
+            ]),
+            PA12: (pa12, 12, Input<Floating>, moder12, AFRH, afrh12, bs12, br12, odr12, idr12, pupdr12, ot12, [
+                AF1: (into_af1, af1,),
+                AF6: (into_af6, af6,),
+                AF7: (into_af7, af7,),
+                AF8: (into_af8, af8,),
+                AF15: (into_af15, af15,),
+            ], [
+                AF2: (into_af2, af2, ["stm32f373", "stm32f378",],),
+                AF5: (into_af5, af5, ["stm32f301", "stm32f303xd", "stm32f303xe", "stm32f318", "stm32f398",],),
+                AF9: (into_af9, af9, ["stm32f302", "stm32f303", "stm32f373", "stm32f378", "stm32f334", "stm32f328", "stm32f358", "stm32f398",],),
+                AF10: (into_af10, af10, ["stm32f302", "stm32f303xb", "stm32f303xc", "stm32f303xd", "stm32f303xe", "stm32f373", "stm32f378", "stm32f358", "stm32f398",],),
+                AF11: (into_af11, af11, ["stm32f301", "stm32f318", "stm32f302", "stm32f303", "stm32f334", "stm32f328", "stm32f358", "stm32f398",],),
+                AF13: (into_af13, af13, ["stm32f334",],),
+                AF14: (into_af14, af14, ["stm32f302", "stm32f303xb", "stm32f303xc",],),
+            ]),
+            PA13: (pa13, 13, Input<Floating>, moder13, AFRH, afrh13, bs13, br13, odr13, idr13, pupdr13, ot13, [
```

Comment:
The default mode of PA13, PA14, PA15, PB3 and PB4 is `AF0`, not `Input<Floating>`.
See #82

Reply:
Ah this seems like a rebase mistake. Will fix it